File: 2.0/java/contacts/removeContactListItems.java
import com.callfire.api.client.CallfireClient;

import java.util.Arrays;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        client.contactListsApi().removeListItems(11646003L, Arrays.asList(22277003L, 23377003L));
    }
}

File: 2.0/csharp/contacts/removeContactListItems.cs
using System.Collections.Generic;
using CallfireApiClient;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        client.ContactListsApi.RemoveListItems(11646003, new List<long> {22277003, 23377003});
    }
}
